src/lib/filter.ts

```diff
@@ -22,6 +22,20 @@ export function shouldSkipRequest(binding: Binding, req: AdmissionRequest, capab
   const { metadata } = srcObject || {};
   const combinedNamespaces = [...namespaces, ...capabilityNamespaces];
 
+  // Delete bindings do not work through admission with DeletionTimestamp
+  if (binding.event.includes(Event.Delete) && binding.filters?.deletionTimestamp) {
+    return true;
+  }
+
+  // Test for deletionTimestamp
+  if (
+    binding.filters &&
+    binding.filters.deletionTimestamp &&
```

Comment:
Could this be `binding.filters?.deletionTimestamp`? It'd just be one less conditional (`&&`) to think about while still achieving the same result, I think.

Reply:
resolved in PR